stackdriver_debugger_logpoint.c

```diff
@@ -151,14 +176,21 @@ void evaluate_logpoint(zend_execute_data *execute_data, stackdriver_debugger_log
     }
     ZVAL_STR(&message->message, m);
 
-    zend_hash_next_index_insert_ptr(STACKDRIVER_DEBUGGER_G(collected_messages), message);
+    if (logpoint->callback) {
+        handle_message_callback(logpoint->callback, message);
```

Comment:
What happens if the callback fails?

Reply:
Makes sense, I'll add a test and have it emit a warning message (and clear the exception).